### PR TITLE
Fixing MDCInkLayer ripple reset

### PR DESCRIPTION
**Pull request by romoore**
* Adding unit tests

### CODE REVIEW

File: components/Ink/src/private/MDCInkLayer+Testing.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCInkLayer.h"
+
+@interface MDCInkLayerRipple : CAShapeLayer
+
+
```
Comment (jgunaratne) [inline components/Ink/src/private/MDCInkLayer+Testing.h]: Extra line

Comment (romoore) [inline components/Ink/src/private/MDCInkLayer+Testing.h]: Agreed, I haven't cleaned the code yet.

File: components/Ink/src/private/MDCInkLayer.m [hunk 2]
```diff
@@ -15,6 +15,7 @@
  */
 
 #import "MDCInkLayer.h"
```
Comment (romoore) [inline components/Ink/src/private/MDCInkLayer.m]: Remove this import, rely on +Testing.h to import the primary header?

**Comment by romoore** [comment]
Looks good enough

**Pull request closed without merging**